convert2rhel/utils.py

```diff
@@ -208,7 +208,7 @@ def log_traceback(debug):
         loggerinst.debug(traceback_str)
     else:
         # Print the traceback to the log file in any way
-        loggerinst.file(traceback_str)
+        loggerinst.file(traceback_str)  # pylint: disable=W1201
```

Comment:
I wonder why is it needed for the `.file` but not `.debug`. Also, the traceback str should be just string with no arguments.
Actually, what pylint reported on the latest master on this line is E1103, not W1201:
https://github.com/oamg/convert2rhel/runs/709656220#step:5:41

Reply:
I'll rebase, several files are changed since I send these PR.